frappe/frappeclient.py

```diff
@@ -179,7 +200,7 @@ def get_doc(self, doctype, name="", filters=None, fields=None):
 		res = self.session.get(self.url + "/api/resource/" + doctype + "/" + name,
 			params=params, verify=self.verify, headers=self.headers)
 
-		return self.post_process(res)
+		return frappe._dict(self.post_process(res))
```

Comment:
Why is this needed?

Reply:
So that contents of the response can be accessed by the '.' (dot) notation rather than using get() everytime.